lib/install.js

```diff
@@ -220,12 +213,12 @@ function install (gyp, argv, callback) {
       // download the tarball, gunzip and extract!
 
       if (tarPath) {
-        var input = fs.createReadStream(tarballUrl)
+        var input = fs.createReadStream(tarPath)
         input.pipe(gunzip).pipe(extracter)
         return
       }
 
-      var req = download(tarballUrl)
+      var req = download(release.tarballUrl)
       if (!req) return
```

Comment:
oh, you know what this is for, it's for the `cb(err)` case inside `download` where it doesn't return a `req` because there was an error setting it up; so this makes perfect sense and the error gets handled in the callback.

leaving it alone